src/mixins/filing-template-mixin.ts

```diff
@@ -831,7 +832,7 @@ export default class FilingTemplateMixin extends Mixins(DateMixin) {
     this.setEntityType(draftFiling.business.legalType || this.getBusinessLegalType)
     this.setLegalName(draftFiling.business.legalName || this.getBusinessLegalName)
     this.setFoundingDate(draftFiling.business.foundingDate || this.getBusinessFoundingDate)
-    this.setBusinessStartDate(this.getBusinessStartDate)
+    this.setBusinessStartDate(draftFiling.business.startDate || this.getBusinessStartDate)
```

Comment:
This is a good fallback.

This means you still need to save business start date in App.vue, right?

Reply:
Yes!

Reply:
OK. Your code in App.vue fits in beautifully with the other code 👍